PipelinesTasks/PRMetrics/buildAndReleaseTask/updaters/parameters.ts

```diff
@@ -99,7 +99,7 @@ export default class Parameters {
     this._taskLibWrapper.debug('* Parameters.initializeBaseSize()')
 
     const convertedValue: number = parseInt(baseSize)
-    if (!convertedValue || convertedValue <= 0) {
+    if (!baseSize?.trim() || !convertedValue || convertedValue <= 0) {
```

Comment:
I don't think these `!baseSize?.trim() ||` parts are required. I removed them in my latest iteration. Applies to the three number inputs below